src/lib/features/change-request-access-service/fake-change-request-access-read-model.ts

```diff
@@ -0,0 +1,15 @@
+import { IChangeRequestAccessReadModel } from './change-request-access-read-model';
+
+export class FakeChangeRequestAccessReadModel
```

Comment:
Do we need the auxiliary function `createFakeChangeRequestAccessService` if we can just call `new FakeChangeRequestAccessReadModel(true)`? The main risk I see is having multiple ways of creating the same object in different places. Normally, I like to keep the creation of objects close to the object definition itself, this could mean:
1. Moving that helper method to this file and making the constructor private
2. Not having the constructor accessible outside change-request-access-service (either private or protected IIRC)

Reply:
I added it for consistency reason but since this graph of objects is trivial for now we can skip it. If we wanted to inject fake access service (for now) then having a function for building a dependency graph is useful. Think of each of those methods as mini DI container for real/fake set of objects with type checking and without magic.

Reply:
I usually like to think about constructors as functions creating an object. Factory functions might be useful when the build is complex or if you get rid of classes and just use functions, but I think we didn't make a decision on that area